tests/suites/constraints/constraints_aws.sh

```diff
@@ -0,0 +1,35 @@
+run_constraints_aws() {
```

Comment:
we don't need an aws test specially
we just want to distinguish between a container scenario vs a vm scenario

we just want to have run_constraints() and run_constraints_lxd()
the case bootstrap_provider would call the lxd verson for lxd and the "normal" one for everything else

Reply:
Are you sure? Many clouds support different constraints, all with different params and the like. I guess we could test just the lowest common denominator though

Reply:
I think that there will be problems with 'instance-type' constraint check, because it is valid only for AWS.

So if we wish to make 'LXD' and 'non-LXD' tests, we need to drop  'instance-type' constraint test.